.github/workflows/release.yml

```
@@ -62,7 +62,9 @@ jobs:
       - name: Generate Notification
         id: notification
         if: steps.changesets.outputs.published == 'true'
-        run: message=$(node scripts/notify/index.js '${{ steps.changesets.outputs.publishedPackages }}') && echo ::set-output name=message::${message//$'\n'/'%0A'}
+        run: |
+          message=$(node scripts/notify/index.js '${{ steps.changesets.outputs.publishedPackages }}') && \
```

```suggestion
          message=$(node scripts/notify/index.js '${{ steps.changesets.outputs.publishedPackages }}')
```

Since this is now multiline, could we remove the `&&` part?

---

Sure!